pkg/kademlia/kademlia_test.go

```diff
@@ -175,7 +177,7 @@ func testNode(t *testing.T, bn []pb.Node) (*Kademlia, *grpc.Server) {
 
 	pb.RegisterNodesServer(grpcServer, s)
 	go func() { _ = grpcServer.Serve(lis) }()
-
+	defer assert.NoError(t, os.RemoveAll("db"))
```

Comment:
It should create the test directory in a temp directory. Also this immediately removes the folder.